app/javascript/packages/document-capture/components/acuant-camera.tsx

```diff
@@ -126,11 +126,14 @@ interface AcuantCameraUICallbacks {
    * Optional frame available callback
    */
   onFrameAvailable?: (response: AcuantDetectedResult) => void;
+  /**
+   * Callback that occurs when there is a failure.
+   */
+  onFailure: (error?: AcuantCaptureFailureError, code?: string) => void;
 }
 
 type AcuantCameraUIStart = (
   callbacks: AcuantCameraUICallbacks,
-  onFailure: AcuantFailureCallback,
```

Comment:
Does it still work for the older version to remove this? Same on line 308 below.

Reply:
Good point.  I updated this in my latest commits.